### ISSUE TITLE
Add support for Eclipse OpenJ9 JVM Heap dump

### ISSUE THREAD

**Issue by Thihup** [javamelody/javamelody#1024]
Hi. I tried taking a Heap dump running on OpenJ9 VM from AdoptOpenJDK (https://adoptopenjdk.net) and it failed.
So, I've looked in the source code and already has a verification for IBM J9 JVM, unfortunately, the `java.vendor` returns `AdoptOpenJDK`. However, I guess it would be possible to use the `java.vm.vendor`, since it returns `Eclipse OpenJ9`

**Comment by evernat** [comment]
Does it work with this code in OpenJ9 ?
```
final Class<?> dumpClass = getClass().getClassLoader().loadClass("com.ibm.jvm.Dump");
final Class<?>[] argTypes = null;
final Method dump = dumpClass.getMethod("HeapDump", argTypes);
final Object[] args = null;
dump.invoke(null, args);
```
Ref: https://github.com/javamelody/javamelody/blob/aa5944aaf4fa2b5db536db899d9ad642477b76fd/javamelody-core/src/main/java/net/bull/javamelody/internal/model/Action.java#L428

Does it work to detect OpenJ9 with something like `System.getProperty("java.vendor").contains("OpenJ9")` ?

**Comment by Thihup** [comment]
> Does it work with this code in OpenJ9 ?
> 
> ```java
> final Class<?> dumpClass = getClass().getClassLoader().loadClass("com.ibm.jvm.Dump");
> final Class<?>[] argTypes = null;
> final Method dump = dumpClass.getMethod("HeapDump", argTypes);
> final Object[] args = null;
> dump.invoke(null, args);
> ```
> 
It seems to work.
> 
> https://github.com/javamelody/javamelody/blob/aa5944aaf4fa2b5db536db899d9ad642477b76fd/javamelody-core/src/main/java/net/bull/javamelody/internal/model/Action.java#L428
> 
> Does it work to detect OpenJ9 with something like `System.getProperty("java.vendor").contains("OpenJ9")` ?

Unfortunately no. 
From the source code, there is only one place where the JAVA_VENDOR is used, so I guess something like this would work:
```java
static final boolean IS_J9_VM = System.getProperty("java.vendor").contains("IBM") || System.getProperty("java.vm.vendor").contains("OpenJ9")
```

**Comment by evernat** [comment]
fixed by 793b4b7 and ready for the next release (1.88).
